Doc/library/fcntl.rst

```diff
@@ -124,6 +124,7 @@ The module defines the following functions:
    .. versionchanged:: next
       Add support of arbitrary :term:`bytes-like objects <bytes-like object>`,
       not only :class:`bytes`.
```

Comment:
You should updated your PR, since this "next" entry became "Python 3.14": https://docs.python.org/dev/library/fcntl.html#fcntl.fcntl